package.json

```
@@ -37,9 +37,10 @@
     "css:lint": "stylelint --config .stylelintrc \"src/**/*.scss\" --cache --cache-location .cache/.stylelintcache",
     "prejs": "yarn js:lint",
     "js": "yarn js:compile",
-    "js:lint": "echo 'To be defined'",
+    "js:lint": "eslint ./src ./examples ./scripts",
```

Root config files should be included too:

```suggestion
    "js:lint": "eslint ./*.js ./src ./examples ./scripts",
```

---

Fixed.